packages/react/src/components/Search/Search.tsx

```diff
@@ -55,6 +55,11 @@ export interface SearchProps extends InputPropsBase {
    */
   disabled?: boolean;
 
+  /**
+   * Specify whether or not ExpandableSearch should render expanded or not
+   */
+  isExpanded?: boolean;
+
```

Comment:
I'd rename this prop to reflect more what it's actually doing within the scope of the `Search` component, like `HideCloseButton`, since `Search` doesn't handle expanded capabilities I don't think it should have any knowledge of it either. And then you can set `HideCloseButton={!isExpanded}` from the `ExpandableSearch` component

Reply:
It's just because all of `ExpandableSearch` props are pulled from `Search:SearchProps`. If I change it, it errors in `ExpandableSearch` and that prop name doesn't make sense in that context.